courseware/data_literacy/course_materials.md

```diff
@@ -0,0 +1,99 @@
+
+**Module 2: [Deep Dive into Data "Set"](https://docs.google.com/document/d/e/2PACX-1vRSLiNxqIQJP3Tp-6eLdz_-Iv5nPp_Q_yS2cY1SbF2wrNbGRsQS26s32fd1gZYrVyarnH8R364G3I3N/pub)**
+
+Introduces students to the idea of a data “set” as well as some basic terms and statistical measures (e.g. mean, standard deviation, etc.) while highlighting important caveats and takeaways when drawing conclusions from these measures.
+
+**Module 3: [Relating Data "Sets"](https://docs.google.com/document/d/e/2PACX-1vQnJHyjrK85SLlSmlFwzoAi2bA8javVv__apXmeq_N_e4LFmqyB7x1I-ghM2Y0PGXy0K8RWd2I_lny7/pub)**
+
+Introduces students to the idea of related data sets and describes how to merge data sets into one large data collection. Real data sets are analyzed in depth to demonstrate the benefits of building a nuanced, complete picture of the world.
+
+**Module 4: [Plotting/Graphing](https://docs.google.com/document/d/e/2PACX-1vQUn3ShN4vdfeXw5o-nXkMHWu9Q6DoyO1Ihk1V0QQ6J3ZaRSB9KhPcgymjvZYA95G86tLQGnsWHNPzd/pub)**
+
+Introduces students to some common plots and graphs and discusses when to use each depending on the data and the context. It also defines and explores various distributions with analysis of their properties and examples from the real world
+
+**Module 5: [Correlations](https://docs.google.com/document/d/e/2PACX-1vTKng32E3mwC6whbEXoFw9UyTHMSH9HH-ySoZ-lRlzB0DTXc72i4VmXuYj0a9LjkakQ-w84IkClQM4q/pub)**
+
+Introduces students to the idea of correlation between two variables and describes how to observe, quantify, and label correlations, with a focus on linear correlations. Strategies for distinguishing real correlations from noise are provided and common pitfalls, such as correlation vs causation, are discussed in depth.
+  
+**Module 6: [Mapping](https://docs.google.com/document/d/e/2PACX-1vQ947FunXunwK32SQeKQHSi7F8P97mQusHh4WjgMKtnp-y_MX_sAGosKUDgX7xaKK1QdpMQCsPXaJKV/pub)**
+
+Introduces students to the idea of geographic data and discusses how to visualize and aggregate geographic data.
+
+**Module 7: Time Series**
+
+Coming soon!
+
+## How To Use
+
+The curriculum is organized in self-contained modules. Instructors (and students) are welcome to consume or work through any of the modules or the components within any module. Each [module](#modules) is available as a detailed document with several pedagogical notes, objectives and an outline/table of contents to allow easy navigation.
+
+To find what you may be looking for, or to use any content in your own courses/curriculum, we recommend the following approach:
+
+1. Read our [about the course page](about_the_course.html) to familiarize yourself with the curriculum objectives which may explain our curriculum design choices.
+
+2. Navigate to the [key themes section](topics.html#key-themes) below. It lists several key themes along with some pointers to specific modules which have the complete details.
```

Comment:
Maybe remove "below" since the key themes section isn't on this page?

Reply:
Done